.github/workflows/dev-image-update.yaml

```diff
@@ -67,6 +67,7 @@ jobs:
           helm_env_prefix: dev
       - name: 'Release Candidate Container Build: Checkout tag for DataBiosphere/jade-data-repo'
```

Comment:
Is the pull because the commit wasn't checked out? If that's the case, I think that in the initial checkout action, if you set `fetch-depth` in that action, it wouldn't require this explicit pull since it pulls down the full git history... I'm actually not sure which would make more sense.

Reply:
Nevermind, caught up offline and it's to reflect changes that have happened upstream and refresh the local git